docs/src/components/AppRouter.js

```diff
@@ -126,15 +126,29 @@ export default function AppRouter() {
           />
         </Route>
         <Route
-          title="Layout"
-          path="/layout"
+          title="Responsive UI"
+          path="/responsive"
```

Comment:
We lose the consistency with the official documentation: https://material.io/guidelines/layout/responsive-ui.html